### DISCUSSION TITLE
part-ii/2020-104/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#310] [Question comments]
Part II 2020 Number Theory: Paper 3, Section I, $1 \mathrm{H}$ 

https://questions.tripos.org/part-ii/2020-104/

**Comment by callumhobbis** [comment]
<details>
<summary>Sketch solution</summary>

All bookwork. For the middle part, note the bases to which $N$ is an Euler pseudoprime form a subgroup of $(\mathbb Z/n\mathbb Z)^*$ (I'm not sure if you need to prove this, but it all comes from the fact that $(b_1b_2)^{\tfrac12(N-1)}=b_1^{\tfrac12(N-1)}b_2^{\tfrac12(N-1)}$ and the Jacobi symbol being multiplicative. Since we have $b_0$, this subgroup isn't the whole group, and thus has index at least 2, i.e. at least half the bases don't work.

The last part is a proof given in lecture 21, which splits into cases: $N$ squarefree and $p^2\mid N$.

</details>
